app/models/kudo.rb

```diff
@@ -1,7 +1,6 @@
 class Kudo < ApplicationRecord
   validates :title, presence: true
   validates :content, presence: true
-  belongs_to :giver, class_name: 'Employee' # , dependent: :kudo
+  belongs_to :giver, class_name: 'Employee'
   belongs_to :receiver, class_name: 'Employee'
-  has_many :company_values, class_name: "CompanyValue", foreign_key: "company_vaule_id"
```

Comment:
Kudo should still depend on the existance on Company Value, make sure to ensure their relation.